<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bitacora</class>
 <widget class="QDialog" name="bitacora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit√°cora</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>801</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar{
alignment: right;
}

QTabBar::tab { min-width:200px; }

QTabBar::tab:!selected{
background-color: #000000;
color:#ffffff;font-size:8pt;
height: 50px;
}
QTabBar::tab:selected{
background-color: #000000;
color:#ffffff;
font-size:12pt;
height: 50px;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Mantenimientos</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>484</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECIONAR UNA RUTINA DE LA LISTA</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_horas">
     <property name="geometry">
      <rect>
       <x>414</x>
       <y>502</y>
       <width>120</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="key_Reschedule">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>492</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#key_Reschedule{ 
background-image: url(:/iconos/screen800x600/iconos/Posponer azul largo.png);
border: none;
background-repeat: none;
background-position: center;
}

#key_Reschedule::pressed{ 
background-image: url(:/iconos/screen800x600/iconos/Posponer blanco.png);
border: none;
background-repeat: none;
background-position: center;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="key_OK">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>490</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#key_OK{ 
background-image: url(:/iconos/screen800x600/iconos/Listo azul.png);
border: none;
background-repeat: none;
background-position: center;
}

#key_OK::pressed{ 
background-image: url(:/iconos/screen800x600/iconos/Listo blanco.png);
border: none;
background-repeat: none;
background-position: center;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>tableWidget_2</zorder>
    <zorder>label_2</zorder>
    <zorder>key_Reschedule</zorder>
    <zorder>key_OK</zorder>
    <zorder>label_horas</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Todas las rutinas</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="top_menu_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_1 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Campana.png);
border: none; 
background-repeat: none;
background-position: center;

 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_3 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Diagrama.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>34</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_5 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Ajustes azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_2 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Bitacora azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>label</zorder>
  <zorder>top_menu_1</zorder>
  <zorder>top_menu_3</zorder>
  <zorder>top_menu_5</zorder>
  <zorder>top_menu_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
